### DISCUSSION TITLE
Module Less not working with defined variable

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#148] [1. Help] [answered]
Hi,

I have added less file as per  https://breezefront.com/docs/styles Module Entry points.

As per attached screenshot, You can see my _default.less file is in module_dir/web/breeze folder 
![image](https://github.com/breezefront/community/assets/132253509/51f9c747-f1fb-47ea-804f-adb90d3e30a8)

And my variables are defined in my custom theme folder.
![image](https://github.com/breezefront/community/assets/132253509/4375a842-de9b-4012-8177-773f948e44b9)

Now, When I am running deploy, It's giving error for undefined variable.
![image](https://github.com/breezefront/community/assets/132253509/d80ca67c-9cee-4cb2-b5d6-366100a2b38f)

Let me know how can I resolve this issue.

Thanks

**Comment by vovayatsyuk** [comment, marked as answer]
On your screenshot, Magento compiles your module with the breeze-blank theme. But your variable is not available in the blank theme, since it's declared in the custom theme.

p.s. You will get the same error if you will add a new variable into the custom luma-based theme, use it in your luma-based module, and then compile the luma theme with your module enabled
